models/stg_amazon_ads__ad_group_history.sql

```diff
@@ -21,12 +21,12 @@ fields as (
 final as (
     
     select 
-        _fivetran_synced,
+        _fivetran_synced, 
```

Comment:
Is the `_fivetran_synced` field necessary to be included in the staging models? I don't think it adds value in our downstream models. We should probably remove this field in the models. 

This is also consistent with our other ad packages as well.

Reply:
Please make this update in the other staging models as well.

Reply:
done